core/src/main/scala/org/apache/spark/scheduler/local/LocalSchedulerBackend.scala

```diff
@@ -76,6 +81,9 @@ private[spark] class LocalEndpoint(
 
     case KillTask(taskId, interruptThread, reason) =>
       executor.killTask(taskId, interruptThread, reason)
+
+    case UpdateDelegationTokens(tokens) =>
+      SparkHadoopUtil.get.addDelegationTokens(tokens, scheduler.conf)
```

Comment:
It would be good to add some info like in `CoarseGrainedExecutorBackend` to increase debug possibilities:
```
      logInfo(s"Received tokens of ${tokenBytes.length} bytes")
```